plugins/genesis-tools/skills/living-docs/SKILL.md

```
@@ -0,0 +1,438 @@
+```
+
+## Entry Points
+
+| Action | Function | Location |
+|--------|----------|----------|
+| Create X | `createX()` | `lib/feature.ts:L24` |
+
+## Gotchas
+
+- [Only non-obvious things that waste time]
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)